airbyte-db/lib/src/main/resources/jobs_database/schema_dump.txt

```diff
@@ -28,6 +28,7 @@ create table "public"."attempts"(
   "created_at" timestamptz(35) null,
   "updated_at" timestamptz(35) null,
   "ended_at" timestamptz(35) null,
+  "temporal_workflow_id" varchar(2147483647) not null default '''''::character varying',
```

Comment:
Without limiting the length of the `VARCHAR`, the max length is really large and unnecessary.

Reply:
oh I didn't realise this was what was happening under the hood. I'm going to put a 256 limit then.